denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/mod_main.dsc

```diff
@@ -21,38 +21,38 @@ mod_command:
     - if <context.args.is_empty>:
       - determine <[arguments]>
     - else if <context.args.size> == 1 && <context.raw_args.ends_with[<&sp>].not>:
-      - determine <[arguments].filter[starts_with[<context.args.first>]]>
+      - determine <[arguments].filter[starts_with[<context.args.get[1]>]]>
   script:
     # -- Hopefully this logic will work & make sense in a few weeks.
     - if <context.args.is_empty>:
       - inventory open d:<inventory[mod_online_inv]>
-    - else if <context.args.first> == version:
+    - else if <context.args.get[1]> == version:
       - narrate "<&6>Adriftus <&e>Moderator Panel"
       - narrate "<&f>Version 2.0.0 - 2020-07-31"
       - narrate "<&f>Scripted by <&b>Kyu#5957"
-    - else if <server.match_offline_player[<context.args.first>]||null> != null:
-      - if <server.match_offline_player[<context.args.first>].name> == <player.name>:
+    - else if <server.match_offline_player[<context.args.get[1]>]||null> != null:
+      - if <server.match_offline_player[<context.args.get[1]>].name> == <player.name>:
         - narrate "<&c>You cannot perform actions on yourself."
         - stop
-      - else if <server.match_offline_player[<context.args.first>].has_permission[mod.staff]>:
+      - else if <server.match_offline_player[<context.args.get[1]>].has_permission[mod.staff]>:
         - narrate "<&c>You cannot perform actions on other staff members."
         - stop
       - else:
         - flag <player> amp_map:!
-        - define uuid <server.match_offline_player[<context.args.first>].uuid>
+        - define uuid <server.match_offline_player[<context.args.get[1]>].uuid>
         # Check if target player is offline
         - if <[uuid].as_player.is_online>:
           # Define YAML ID
           - define id global.player.<[uuid]>
         - else:
           # Define directory and YAML ID
-          - define dir data/players/<[uuid]>.yml
+          - define dir data/global/players/<[uuid]>.yml
```

Comment:
this is the only viable change here - every other change is a step back from consistency updates made recently